app/assets/javascripts/channels/chat.js

```diff
@@ -0,0 +1,21 @@
+App.chat = App.cable.subscriptions.create("ChatChannel", {
+  connected: function() {
+    // Called when the subscription is ready for use on the server
+  },
+
+  disconnected: function() {
+    // Called when the subscription has been terminated by the server
+  },
+
+  received: function(data) {
+    console.log(data)
+    console.log("Hello I am working")
```

Comment:
is this meant to be here?

Reply:
No I will remove that and add the next pull request should have test coverage if we can figure it out.